.github/workflows/main.yml

```diff
@@ -8,12 +8,21 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04]
+        version: ['latest', '1.22.3']
     steps:
       - uses: actions/checkout@v3
       - uses: ./
         with:
           ddevDir: tests/fixtures/ddevProj1
           autostart: false
+          version: ${{ matrix.version }}
+      - name: ddev version
+        run: |
+          if [[ ${{ matrix.version }} == '1.22.3' ]]; then
+            test $(ddev --version) == 'ddev version v1.22.3'
+          else
+            test $(ddev --version) != 'ddev version v1.22.3'
+          fi
```

Comment:
This seems too fragile; can we do something less specific? What's the purpose of this exact stanza?

Reply:
When using latest as the ddev version, then the version will change over time. This basically checks that if the version is specified, it matches, and if not, it can never match (now that 1.22.4 is out).

I'm open to suggestions here!

Reply:
That looks good to me as a test.